man/fi_gni.7.md

```diff
@@ -256,6 +256,14 @@ GNIX_FAB_RQ_NAMO_FAX (Fetch AND and XOR) and GNIX_FAB_RQ_NAMO_FAX_S
 
 #NOTES
 
+The default address format is FI_ADDR_GNI. This is the only address format
+used within the GNI provider for message passing. FI_ADDR_STR is always
+parsed and converted to FI_ADDR_GNI for use within the GNI provider.
+
+*FI_ADDR_STR* is formatted as follows:
+gni[;[node][;service]];GNIX_AV_STR_ADDR_VERSION:device_addr:cdm_id:name_type
+:cm_nic_cdm_id:cookie:rx_ctx_cnt
```

Comment:
okay, either mixing or not is fine with me. FWIW, I was trying to separate the `struct gnix_ep_name` fields from the other fields by using ';' and ':'.